### DISCUSSION TITLE
RDP Support

### DISCUSSION THREAD

**Discussion by leadOnGmbH** [rustdesk/rustdesk#9882] [General]
Would be nice to have native RDP Support (like Microsofts windows app - formerly Remote Desktop app) to Windows Workstation/Server without installing the RustDesk Client on the machines.
Just input the ip-address of the remote client (instead of the RustDesk ID) and it will open the RDP session.

**Comment by rustdesk** [comment]
No, we won't do RDP, this is out of scope. But you can access RDP via our port forwarding with RustDesk.
https://github.com/rustdesk/rustdesk/wiki/FAQ#how-to-use-rdp-via-tcp-tunneling-port-forward-
